src/PIL/TiffImagePlugin.py

```diff
@@ -1275,6 +1275,13 @@ def load(self) -> Image.core.PixelAccess | None:
             return self._load_libtiff()
         return super().load()
 
+    @property
+    def size(self) -> tuple[int, int]:
+        if hasattr(self, "tag_v2"):
```

Comment:
There is a cases after copying or pickling, when the class is actually `TiffImageFile`, but it is loaded and doesn't have `tag_v2` attribute